services/wallet/transfer/commands_sequential_test.go

```diff
@@ -713,6 +714,87 @@ func (tc *TestClient) BlockNumber(ctx context.Context) (uint64, error) {
 	}
 	return 0, nil
 }
+
+func (tc *TestClient) FeeHistory(ctx context.Context, blockCount uint64, lastBlock *big.Int, rewardPercentiles []float64) (*ethereum.FeeHistory, error) {
+	err := tc.countAndlog("FeeHistory")
+	if err != nil {
+		return nil, err
+	}
+	return nil, nil
```

Comment:
🤔 
```suggestion
	return nil, err
```


(and same for all functions below)

Reply:
this is what I get for copypasting